### PR TITLE
dark magic v0.1.0: pipx

### PR DESCRIPTION
**Pull request by jhheider**

### CODE REVIEW

File: src/hooks/useDarkMagic.ts [hunk 1]
```diff
@@ -28,3 +29,17 @@ const npx = async (arg0: string | undefined) => {
 
   return undefined
 }
+
+const pipx = async (arg0: string | undefined) => {
+  const res = await useFetch(`https://pypi.org/pypi/${arg0}/json`)
```
Comment (mxcl) [inline src/hooks/useDarkMagic.ts:34]: is this how pipx does it?

Comment (jhheider) [reply src/hooks/useDarkMagic.ts:34]: ChatGPT says so.

Comment (jhheider) [reply src/hooks/useDarkMagic.ts:34]: Actually ChatGPT said XML-RPC, but if I wanted JSON that's the variant.

Comment (mxcl) [reply src/hooks/useDarkMagic.ts:34]: I'll cross fingers chat GPT wasn't hallucinating.

**Review by mxcl — APPROVED** [review]

**Comment by jhheider** [comment]
It seemed to work ;)

**Comment by mxcl** [comment]
works vs. works exactly like pipx is not necessarily the same thing ofc

**Pull request merged**